src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,123 @@
+import { useContext, useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { PeopleContext } from './PeopleContext';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import classNames from 'classnames';
+import { PersonLink } from './PersonLink';
+import React from 'react';
+
+export const PeoplePage: React.FC = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [error, setError] = useState<boolean>(false);
+  const { state, dispatch } = useContext(PeopleContext);
+  const { people } = state;
```

Comment:
if you don't use it somewhere else the context is redundant

Reply:
not fixed